tests/bluetooth/host/hci/CMakeLists.txt

```
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(test_host_hci)
+
+FILE(GLOB app_sources src/*.c)
```

```suggestion
FILE(GLOB CONFIGURE_DEPENDS app_sources src/*.c)
```

https://cmake.org/cmake/help/latest/command/file.html#glob

---

How about just not using `GLOB`? :)